src/main.py

```diff
@@ -318,5 +342,79 @@ async def getUser(ctx, comp):
         return await ctx.bot.fetch_user(id)
     except:
         return None
-
-bot.run(credentials['token'])
\ No newline at end of file
+#
+# @bot.command(name="TTT")
+# async def neuTrivia(ctx,competitor):
+#     # msg = await ctx.send("{0} has started a game of MYSTERYGAME".format(ctx.author.name))
+#     # await msg.add_reaction(emojiLocal['STONKSUP'])
+#     # await msg.add_reaction(emojiLocal['STONKSDOWN'])
+#     # determining
+#     game_start_embed = utils.get_embed(title="TIC TAC TOE",desc="{0} has challenged {1} to a game of TIC-TAC-TOE".format(ctx.author.name,competitor),color=discord.Color.blue())
+#     game_start_embed.add_field(name="Accept",value="React with your chosen emojis to accept")
+#     game_start_embed.add_field(name="Deny", value="Ignore if you want to deny")
+#
+#     competitor = await getUser(ctx, competitor)
+#     msg = await ctx.send(embed=game_start_embed)
+#
+#     # Get emojis for playing the game.
+#
+#     # Check for users accepting the match
+#     user_one = False
+#     user_two = False
+#     emoji_one , emoji_two = None,None
+#     def check(reaction, user):
```

Comment:
Can this game logic be moved to games/TicTacToe.py ?
Let's not bring too much of the logic to the main file ( although it needs to be refactored ).